src/shared/components/ui/Spacer/Spacer.js

```diff
@@ -12,13 +10,18 @@ const cssClass = styles(scss)
  * @param {string} props.space - space size
  * @return {object} An object of children element
  */
-const spacer = ({ className, space }) => (
-  <div className={cssClass(className, `spacer--${space}`)} />
-)
+const Spacer = ({ className, space }) => {
+  const spacerClasses = useStyles({
+    [styles[className]]: true,
```

Comment:
a więc tak className prop daje defaultowo klasę spacer, ale służy też do przekazania randomowej klasy z zewnątrz, jeżeli przekażesz w tablicy klasę dodatkową czyli zostawisz spacer + coś jeszcze to się wysypie bo pierwszy argument useStyles szuka w stylach modułowych. Do hooka useStyles został dodany drugi parametr w którym możesz przekazać całe className z klasami zewnętrznymi a klasę modułową podać jawnie jako pole `styles['spacer]: true` w pierwszym argumencie no czyli tym obiekcie. Zrobiliśmy to tak, żeby rozdzielić style modułowe od zewnętrznych randomowych klas. Warto też dodać className defaultowo jako pusty string żeby nie została klasa 'spacer'